terraform/eks/e2e/main.tf

```diff
@@ -34,9 +34,9 @@ resource "aws_eks_node_group" "this" {
   subnet_ids      = module.basic_components.public_subnet_ids
 
   scaling_config {
-    desired_size = 2
-    max_size     = 2
-    min_size     = 2
```

Comment:
So we are _always_ generating a single node group? I think it would be useful if are able to configure it